src/server/Server.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SERVER_H
+#define SERVER_H
+
+#include <fcntl.h>
+#include <netdb.h>
+#include <sys/poll.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <cstring>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include "../common/log.h"
+#include "../common/magicNumber.h"
+
+extern bool isServerRunning_g;
+
```

Comment:
Remember to declare extern errno

Reply:
errno is a macro, not declarable.. added include for it though now